packages/tabs/src/lib/dom.js

```diff
@@ -38,62 +38,101 @@ export const initUI = Store => ({ tabs, panels }) => {
     });
 };
 
-const getPreviousIndex = ({ activeIndex, tabs }) => activeIndex === 0 ? tabs.length - 1 : activeIndex - 1;
+const getPreviousTabIndex = ({ activeTabIndex, tabs }) => activeTabIndex === 0 ? tabs.length - 1 : activeTabIndex - 1;
 
-const getNextIndex = ({ activeIndex, tabs }) => activeIndex === tabs.length - 1 ? 0 : activeIndex + 1;
+const getNextTabIndex = ({ activeTabIndex, tabs }) => activeTabIndex === tabs.length - 1 ? 0 : activeTabIndex + 1;
 
 const initListeners = (tab, nextIndex, Store) => {
+    const isManualActivation = Store.getState().settings.activation === 'manual';

```

Comment:
Would it be worth adding the string "manual" to ./constants.js as an ACTIVATION_MODE or something, together with default="auto"? This would also provide an opportunity to document the use you describe in the feature.

Reply:
Yes, good plan - will do that!

Reply:
Updated in [https://github.com/stormid/components/pull/186/commits/89e485dc9f7c71ab22c0d24d519d9537bfc17983](https://github.com/stormid/components/pull/186/commits/89e485dc9f7c71ab22c0d24d519d9537bfc17983)